src/Platform.Handlers/samples/Sample/Controls/Slider.cs

```diff
@@ -81,5 +80,41 @@ void ISlider.DragCompleted()
 				DragCompleted?.Invoke();
 			}
 		}
+
+		public static double Clamp(double self, double min, double max)
```

Comment:
These are internal inside our handler code. I didn't want to just make those public willy nilly